scripts/checkov.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Copyright 2024-2025 New Vector Ltd
+#
+# SPDX-License-Identifier: AGPL-3.0-only
+
+set -euo pipefail
+
+checkov --version
+export HELM_NAMESPACE=ess
+workdir="$PWD"
+matrix_stack_path="$workdir/charts/matrix-stack"
+for checkov_values in charts/matrix-stack/ci/*checkov*values.yaml; do
```

Comment:
I think this script shouldn't do the looping but take a specific values file and test that